openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/CustomNode.component.tsx

```diff
@@ -213,23 +229,26 @@ const CustomNode = (props: NodeProps) => {
       ) : null}
       <section
         className={classNames('tw--mx-3 tw-px-3', {
-          'tw-h-36 tw-overflow-y-auto': columns?.length,
+          // 'tw-h-36 tw-overflow-y-auto': !isEmpty(columns),
```

Comment:
Any reason we have removed this?

Reply:
earlier in E2E lineage we were showing columns only clicking on the + icon, but now as we have column lineage we are showing all the column directly so user can connect the C2C lineage

Reply:
Let's remove the line itself if not needed